49.md

```diff
@@ -0,0 +1,83 @@
+NIP-49
+======
+
+Nostr Wallet Auth
+--------------------
+
+`draft` `optional` `author:benthecarman`
+
+## Rationale
+
+[Nostr Wallet Connect](47.md) is a protocol for connecting a Nostr client to a remote Lightning wallet. It is great but
+hinges on having the user copy-paste a wallet connection URI into their client. This can be a UX hurdle and often has
+the user handling sensitive information that they may not understand.
+
+As well, Nostr Wallet Connect has lots of different functions that a wallet may or may not support or a service may or
+may not need. This either runs into silent failures or requires the user to understand what functions their wallet
+supports and what functions the service needs.
+
+Finally, Nostr Wallet Connect requires the wallet service to generate both key pairs and the service to use these. This
+limits the key management strategies that a service can use and often leads to suboptimal key management.
+
+This NIP proposes a new protocol that solves these problems by having the client and service generate a NWC connection
+URI together. This URI is then used to connect the client and service.
+
+## Theory of Operation
+
+1. The user wants to connect their wallet to a service. The service provides a QR code or other means of connecting to
+   the service. The QR code contains a `nip49://` URI with a `service_key` parameter that contains the service's public
```

Comment:
`nip49://` or `nostr+walletauth://` (used below)? Are they the same? Or are two URIs being defined?

Reply:
woops, missed that